Solutions/GoogleCloudPlatformIAM/Data Connectors/IAM_CCP/IamDefinition.json

```diff
@@ -0,0 +1,113 @@
+                    "legend": "Google IAM logs",
+                    "baseQuery": "{{graphQueriesTableName}}"
+                }
+            ],
+            "sampleQueries": [
+                {
+                    "description" : "Get sample of GcpIAM logs",
+                    "query": "{{graphQueriesTableName}}\n| take 10"
+                }
+            ],
+            "dataTypes": [
+                {
+                    "name": "{{graphQueriesTableName}}",
+                    "lastDataReceivedQuery": "{{graphQueriesTableName}}\n | summarize Time = max(TimeGenerated)\n  | where isnotempty(Time)"
+                }
+            ],
+            "connectivityCriteria": [
+                {
+                    "type": "HasDataConnectors"
+                    
+                }
+            ],
+            "availability": {
+                "status": 1,
+                "isPreview": false
+            },
+            "permissions": {
+                "resourceProvider": [
+                    {
+                        "provider": "Microsoft.OperationalInsights/workspaces",
+                        "permissionsDisplayText": "Read and Write permissions are required.",
+                        "providerDisplayName": "Workspace",
+                        "scope": "Workspace",
+                        "requiredPermissions": {
+                            "read": true,
+                            "write": true,
+                            "delete": true,
+                            "action": false
+                        }
+                    }
+                ]
+            },
+            "instructionSteps": [
+                {
+                    "instructions": [
+                        {
+                            "type": "Markdown",
+                            "parameters": {
+                              "content": ">**NOTE:** If both Azure Function and CCP connector are running parallelly, duplicate data is populated in the tables."
+                            }
+                          },
+                        {
+                            "type": "MarkdownControlEnvBased",
+                            "parameters": {
+                                "prodScript": "#### 1. Setup the GCP environment \n Ensure to have the following resources from the GCP Console:\n Project ID, Project Name, GCP Subscription name for the project, Workload Identity Pool ID, Workspace Identity Provider ID, and a Service Account to establish the connection.\n For more information, refer the [Connector tutorial](https://github.com/v-hkopparala/v-hkopparala/blob/main/README.md) for log setup and authentication setup tutorial.\n Log set up script: [Click Here](https://github.com/v-hkopparala/v-hkopparala/blob/main/CCPIAMLOGS%201.tf)\nAuthentication set up script: [Click here](https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/GCP/Terraform/sentinel_resources_creation/GCPInitialAuthenticationSetup)",
```

Comment:
https://github.com/v-hkopparala/v-hkopparala/blob/main/README.md 

Link not accessible